experiments/ClimaEarth/setup_run.jl

```diff
@@ -529,12 +529,10 @@ function CoupledSimulation(config_dict::AbstractDict)
         # called below at reinit). Turbulent fluxes in atmos are updated in `update_model_sims`.
         Interfacer.update_field!(atmos_sim, Val(:surface_temperature), cs.fields)
 
-
         # 5.reinitialize models + radiative flux: prognostic states and time are set to their initial conditions. For atmos, this also triggers the callbacks and sets a nonzero radiation flux (given the new sfc_conditions)
         FieldExchanger.reinit_model_sims!(cs.model_sims)
 
         # 6.update all fluxes.
-        FieldExchanger.import_atmos_fields!(cs.fields, cs.model_sims)
```

Comment:
I don't know why we were importing atmos fields here. Maybe because combined fluxes were computed in atmos and we needed to retrieve them. But with partitioned fluxes I don't see a need for it